### PR TITLE
Correction to xgboost requirements files [skip ci]

### PR DESCRIPTION
**Pull request by ZiyueXu77**
Fixes # .

### Description

Add item and corrections to the xgboost requirement files

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: examples/advanced/xgboost/histogram-based/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 nvflare>=2.3.0rc3
+pandas
 xgboost==1.7.0rc1
```
Comment (YuanTingHsieh) [inline examples/advanced/xgboost/histogram-based/requirements.txt:3]: Can you try if just put xgboost>=1.7.0 work?

File: examples/advanced/xgboost/tree-based/requirements.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 nvflare>=2.3.0rc3
 pandas
 xgboost
```
Comment (YuanTingHsieh) [inline examples/advanced/xgboost/tree-based/requirements.txt:3]: Can you try if xgboost >=1.7.0 work

Comment (ZiyueXu77) [reply examples/advanced/xgboost/tree-based/requirements.txt:3]: tested and updated

Comment (YuanTingHsieh) [inline examples/advanced/xgboost/tree-based/requirements.txt:3]: we don't need >= 1.7.0 here?

**Pull request merged**